src/components/seeds/database/index.tsx

```diff
@@ -712,15 +716,17 @@ export default function Database(props: DatabaseProps): JSX.Element {
                       }}
                     >
                       {isOnboarded && availableFieldOptions && fieldOptions && (
-                        <Filters
-                          filters={searchCriteria}
-                          availableValues={availableFieldOptions}
-                          allValues={fieldOptions}
-                          columns={filterColumns}
-                          searchColumns={searchTermColumns}
-                          preExpFilterColumn={preExpFilterColumn}
-                          onChange={onFilterChange}
-                        />
+                        <>
```

Comment:
need this?

Reply:
Hmmm... not sure what that's for 😅 . Removed